packages/flutter/lib/fix_data.yaml

```diff
@@ -11,6 +11,168 @@
 version: 1
 transforms:
 
+  # Changes made in https://github.com/flutter/flutter/pull/61648
+  - title: 'Migrate to autovalidateMode'
+    date: 2020-09-02
+    element:
+      uris: [ 'widgets.dart' ]
```

Comment:
Isn't this also accessible through material.dart and cupertino.dart?

Reply:
We should do an audit of this file. `material.dart` and `cupertino.dart` export `widget.dart`, so whenever we have something in `widget.dart`, those other two need to be mentioned as well.

@brianegan We talked about this before, I really wish there was an easier way to fill in the uris field.

Reply:
(this also applies to some of the other fixes below and should probably be covered with tests as well)

Reply:
I'll update this, and then open another PR that audits the rest and adds tests for those as well.